src/hooks/useActiveWorkspaceFromNavigationState.ts

```diff
@@ -0,0 +1,13 @@
+import {useNavigationState} from '@react-navigation/native';
+import type {BottomTabNavigatorParamList} from '@libs/Navigation/types';
+
+/**
+ *  Get the currently selected policy ID stored in the navigation state. This hook should only be called only from screens in BottomTab.
+ */
+function useActiveWorkspaceFromNavigationState() {
+    const activeWorkpsaceID = useNavigationState<BottomTabNavigatorParamList, string | undefined>((state) => state.routes.at(-1)?.params?.policyID);
```

Comment:
This line should have a comment with explanation. Thank you for checking it out! We always store the last value of policyID in the last route of BottomTab, so that's why we need to use `state.routes.at(-1)?.params?.policyID` to obtain the policyID of the active workspace. Furthermore, it's important to remember that this hook should be called only from BottomTab screens as it's mentioned in the comment above the function.